src/redux/types.ts

```diff
@@ -0,0 +1,146 @@
+import { TABS, DROPDOWN_OPTIONS_DEFINITION } from 'config';
+
+export type Movie = {
+  id: number;
+  title: string;
+  tagline: string;
+  vote_average: number;
+  vote_count: number;
+  release_date: string;
+  poster_path: string;
+  overview: string;
+  budget: number;
+  revenue: number;
+  genres: Array<string>;
+  runtime: number;
+};
+
+export interface RootReducer {
+  activeTab: typeof TABS[number];
+  sortBy: keyof typeof DROPDOWN_OPTIONS_DEFINITION;
+  movies: Movie[];
+  isMoviesLoading: boolean;
+  movieDetails: Movie | null;
+  error: string;
+}
+
+// SET_ACTIVE_TAB
+export const SET_ACTIVE_TAB = 'SET_ACTIVE_TAB';
```

Comment:
I would not mix redux and movie types here.

Reply:
but it is redux types